examples/create_rosa_cluster/create_rosa_sts_cluster/basic_sts/cluster/variables.tf

```diff
@@ -10,4 +10,4 @@ variable operator_role_prefix {
 variable url {
     type = string
     default = "https://api.openshift.com"
-}
+}
```

Comment:
not sure that "basic_sts" is appropriate.. there is a common terminology for a non-BYO model?

Reply:
So, there isn't right now. I'll check

Reply:
Went with classic after talking to Aaren